lib/termtable.h

```diff
@@ -278,6 +278,18 @@ void ttable_rowseps(struct ttable *tt, unsigned int row,
  */
 char *ttable_dump(struct ttable *tt, const char *newline);
 
+/**
+ * Dumps the table, and frees the table
+ *
+ * @parm  vty - The terminal to print to
+ * @param tt - double pointer to the table to dump, this pointer
+ *             is freed.
+ * @param newline the desired newline sequence to use, null terminated.
+ * @param display_if_no_rows The string to display if nrows is 0
```

Comment:
This description is incorrect as it will currently (perhaps erroneously) display the string if nrows is 0 *or* 1.